node_modules/gh-series/tests/test-crud.js

```diff
@@ -246,6 +265,43 @@ describe('Series', function() {
                 });
             });
         });
+
+        /**
+         * Test that verifies that the organisational unit a series is borrowed from is indicated when getting a series and including the organisational units
```

Comment:
As this is already under the `Getting a series` block, this can be changed to `Test that verifies that the organisational unit a series is borrowed from is returned when including the organisational units`